packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -679,11 +709,12 @@ public void run() {
 
   /*package*/ void removeViewWithSubviewClippingEnabled(View view) {
     UiThreadUtil.assertOnUiThread();
-
     Assertions.assertCondition(mRemoveClippedSubviews);
     Assertions.assertNotNull(mClippingRect);
     View[] childArray = Assertions.assertNotNull(mAllChildren);
+
     view.removeOnLayoutChangeListener(mChildrenLayoutChangeListener);
+
```

Comment:
Can undo these if there aren't going to be any other changes here

Reply:
+1, done here: https://github.com/facebook/react-native/pull/47634/commits/c14e0268fcc10a78bb60fa65a274f34b14e808c2, that's just merge artefact.